src/mesh/logical_location.hpp

```diff
@@ -0,0 +1,276 @@
+//========================================================================================
+// Athena++ astrophysical MHD code
+// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+// Licensed under the 3-clause BSD License, see LICENSE file for details
```

Comment:
Is this still valid?

Reply:
I am not sure, I left it in since there is still some code that presumably came from Athena++, although it is a small fraction of the file.

Reply:
I am just going to leave it.